meson.build

```diff
@@ -463,6 +463,42 @@ endif
 
 
 
+###############################################################
+# Library: Tcl (for pltcl)
+
+# tclConfig.sh
+#
+# NB: tclConfig.sh is used in autoconf build for getting
+# TCL_SHARED_BUILD, TCL_INCLUDE_SPEC, TCL_LIBS and TCL_LIB_SPEC
+# variables. That is not implemented in meson build.
+###############################################################
+
+tcl_dep = dependency('', required : false)
+tclopt = get_option('pltcl')
+tcl_specified = get_option('tcl_version')
+if not tclopt.disabled()
+  tcl = 'tcl'
+
+  if(tcl_specified != 'none')
```

Comment:
I suggest changing the default of the 'tcl_version' option to 'tcl' instead of 'none'. Then you don't need the branch here?